Packs/PrismaCloud/LayoutRules/Prisma_Cloud_Network_API_and_Anomaly.json

```diff
@@ -0,0 +1,32 @@
+{
+  "rule_id": "Prisma_Cloud_Network_API_and_Anomaly",
+  "layout_id": "Prisma Cloud - Network Incident Layout",
```

Comment:
Please change the layout_id accordingly to the new layout name 'Prisma Cloud - Network API and Anomaly Alerts'.